src/Models/Account.js

```diff
@@ -246,6 +246,19 @@ class Account {
         return responseData;
     }
 
+    async reportSSOtoken(ssoDomain, tokenData, retire = false) {
+        const body = new FormData();
+
+        body.append('token', tokenData.token);
+        body.append('delete', retire ? 1 : 0);
+
+        fetch(this.config.API.reportSSOtoken(ssoDomain), {
```

Comment:
The method is async, the result of `fetch` needs to be `await`-ed and then the response checked for errors.
```suggestion
        const response = await fetch(this.config.API.reportSSOtoken(ssoDomain), {
          ...
        }
        
        checkStatus(response);
        
        return await response.json();
```